src/utils/query.js

```diff
@@ -85,14 +88,15 @@ module.exports = {
     const shouldSecondarySortOnId = params.paginatedField !== '_id';
 
     if (shouldSecondarySortOnId) {
+      const field = params.sortCaseInsensitive ? '__lc' : params.paginatedField;
       return {
-        [params.paginatedField]: sortDir,
+        [field]: sortDir,
         _id: sortDir,
       };
     }
 
     return {
-      [params.paginatedField]: sortDir,
+      _id: sortDir,
```

Comment:
Typo?

```suggestion
      [field]: sortDir,
```

Reply:
No: `params.paginatedField == '_id'` in this branch. I thought this was more explicit. I'll reverse the logic to make it clearer.